pages.ru/common/dotnet-build.md

```diff
@@ -0,0 +1,32 @@
+# dotnet build
+
+> Собирает приложение .NET и все его зависимости.
+> Больше информации: <https://docs.microsoft.com/dotnet/core/tools/dotnet-build>.
+
+- Скомпилировать проект или решение в текущей директории:
+
+`dotnet build`
+
+- Скомпилировать проект или решение .NET в режиме debug:
+
+`dotnet build {{путь/до/проекта_или_решения}}`
+
+- Скомпилировать в режиме release:
+
+`dotnet build`
```

Comment:
As author of the English page: there is a bug :upside_down_face:. This is the correct example for the description:
```suggestion
`dotnet build --configuration Release`
```
Can you include the fix to the English page in this PR?

Reply:
Sure, just fixed it in both original and translation.